404.html

```diff
@@ -21,6 +21,21 @@
                     window.location.href = "/";
                 }
             }, 1000);
+
+            // Check if a phone or iPad
+            if (window.innerWidth <= 700 || /iPad|Tablet/i.test(navigator.userAgent) || (window.innerWidth > 700 && window.innerWidth <= 1024)) {
+                document.body.classList.add('mobile');
```

Comment:
I think this is something that would much better be done with an @media query in CSS

Reply:
Thats a good idea. This is just janky in the first place

Reply:
> I think this is something that would much better be done with an @media query in CSS

Yeah, from what I'm reading, it's not possible to change HTML text without JS. The @media query can only change the state or visual properties. Yes it's possible to change text visually but its more janky and doesn't work with accessibility options like screenreader

Reply:
I stand corrected. I apologize, It's just hiding and showing text thats already there. I'll work on it